Imagekit/Models/UploadTransformation.cs

```diff
@@ -0,0 +1,47 @@
+namespace Imagekit.Models
+{
+    using System.Collections.Generic;
+
+    public class PostTransformation
+    {
+        public string type { get; set; }
+        public string value { get; set; }
+    }
```

Comment:
Documentation for pre and post transform missing in README.md

Reply:
done